w3d_translator/classes/Color.py

```diff
@@ -1,12 +1,26 @@
 import xml.etree.ElementTree as ET
+from dataclasses import dataclass, field
+
+from .XmlDataClass import XmlDataClass
```

Comment:
even with classes, I _think_ the convention is to always name python packages/files with snake case

Reply:
Looks like lowercase for packages/modules and TitleCase for classes. I upsated the `__init__.py` for `classes` to reflect this.

https://peps.python.org/pep-0008/#:~:text=Package%20and%20Module%20Names,-Modules%20should%20have&text=Python%20packages%20should%20also%20have,use%20of%20underscores%20is%20discouraged.
https://peps.python.org/pep-0008/#class-names

Reply:
yup, exactly, so I'd expect this to read `from .xml_data_class import XmlDataClass` (I saw the thing on discouraging underscores and I think with something this long, the underscore is worth it)